src/app/mod.rs

```diff
@@ -30,6 +31,8 @@ use errors::Result as ClapResult;
 pub use self::settings::AppSettings;
 use completions::Shell;
 
+const DEFAULT_ARGS_ENV_VAR: &'static str = "VAR";
```

Comment:
I think we can do away with a default variable name. The user typing `None` is just as if they'd typed the env var name `VAR`